src/mysqlsh_helpers.py

```diff
@@ -336,7 +339,7 @@ def _run_mysqlsh_script(self, script: str) -> None:
             # Specify python as this is not the default in the deb version
             # of the mysql-shell snap
             command = [MySQL.get_mysqlsh_bin(), "--no-wizard", "--python", "-f", _file.name]
-            subprocess.check_output(command, stderr=subprocess.PIPE)
+            return subprocess.check_output(command, stderr=subprocess.PIPE).decode("utf-8")
```

Comment:
Why add the return? I couldn't see where is being used.

Reply:
Ah, this should have been introduced in a future PR. Removed for the time being in fc409d658a9d226f29dd0b36a2723615e638dd98